implementation.md

```diff
@@ -2,4 +2,61 @@
 layout: default
 ---
 
-Work in progress. 
+# Sinopia Technology Implementation
+The Sinopia stack uses [Circle-ci][CI] for continuous integration to
+validate the code syntax using [eslint](https://eslint.org/) and test the code base using a combination of
+[Jest](https://jestjs.io/), [Enzyme](https://github.com/airbnb/enzyme), and
+[Puppeteer](https://developers.google.com/web/tools/puppeteer/). [Circle-ci][CI] also
```

Comment:
Two more of those link-not-link vestiges here.

Reply:
Fixed in latest commit.